backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/llvm/LlvmUtils.kt

```diff
@@ -110,6 +110,7 @@ internal val int64Type get() = LLVMInt64TypeInContext(llvmContext)!!
 internal val int8TypePtr get() = pointerType(int8Type)
 internal val floatType get() = LLVMFloatTypeInContext(llvmContext)!!
 internal val doubleType get() = LLVMDoubleTypeInContext(llvmContext)!!
+internal val vectorType get() = LLVMVectorType(LLVMFloatTypeInContext(llvmContext), 4)!!
```

Comment:
`floatType` can be used instead of `LLVMFloatTypeInContext(llvmContext)`.

Reply:
Done.